swarm/pss/pss.go

```diff
@@ -71,6 +72,7 @@ type PssParams struct {
 	CacheTTL            time.Duration
 	privateKey          *ecdsa.PrivateKey
 	SymKeyCacheCapacity int
+	AllowRaw            bool
```

Comment:
We should start to document the params in structs like this one. Please add a comment next to `AllowRaw` explaining its purpose and usage.